### PR TITLE
Importer Basics

### PR DESCRIPTION
**Pull request by jonuy**
#### What's this PR do?

Provides foundation for importing the SMS game json config files and converting them into Twine stories. Currently only supports the story config passage and regular DS passages.

Also some cleanup with the project name and fields in the package.json.
###### storylistview.js

The file being imported is read here. I added pulling the filename and extension in order to know if it should be processed as one of our .json config files instead.
###### app.js

Basically just acts as a bridge to importsms. Seems like the only reason I have this is to stay consistent with how the original `app.importFile` worked.
###### importsms.js

Reads through all a config object's keys and creates corresponding passages to display in Twine.
#### How should this be manually tested?

I tested with this file: https://gist.github.com/jonuy/88a949644e3eb437da0e

Import it using the "Import From File" button on the Intertwine homepage. Verify that 4 passages get created with data and are connected.

@tongxiang

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ var TwineApp = Backbone.Marionette.Application.extend(
    @property name
   **/
 
-  name: 'Twine',
+  name: 'Intertwine',
```
Comment (tongxiang) [inline js/app.js:20]: :santa: We made it!!!! :octopus:

File: js/importsms.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        story,
+        storyConfig,
+        pIndex,
+        passages,
+        keys,
+        i;
+
+    data = JSON.parse(strData);
+
+    allStories = StoryCollection.all();
```
Comment (tongxiang) [inline js/importsms.js]: Why is it necessary to run the `StoryCollection.all()` function to create stories?

Comment (jonuy) [inline js/importsms.js]: Good question. I have no idea. I just tried it with `new StoryCollection()` and `new PassageCollection()` instead and it seems to work. I'll submit a change in a sec.

**Comment by tongxiang** [comment]
:+1:

**Pull request merged**